src/gtimelog/main.py

```diff
@@ -690,6 +691,7 @@ def load_settings(self):
             self.gsettings.set_string('list-email', old_settings.email)
             self.gsettings.set_string('report-style', old_settings.report_style)
             self.gsettings.set_string('task-list-url', old_settings.task_list_url)
+            self.gsettings.set_string('remote-auth-header', old_settings.auth_header)
```

Comment:
This is code to migrate the old .gtimelogrc configuration file to gsettings.  Was there ever a gtimelog fork that supported an auth-header setting in .gtimelogrc?  If not, this shouldn't be here.